packages/eslint-config/react.js

```diff
@@ -117,6 +117,13 @@ const config = [
             ],
         },
     },
+    {
+        files: ["**/*.ts", "**/*.tsx"],
+        ignores: ["**/*.test.ts", "**/*.test.tsx", "**/*.spec.ts", "**/*.spec.tsx"],
+        rules: {
+            "react/jsx-no-literals": "error",
```

Comment:
Open Question:

- Do we want to ship this to main/next?
- Do we want to introduce something like a `react-next.js` what is used in demo repo but not in projects? As soon as we ship next major version, we are resetting `react.js` to `react-next.js`?

Reply:
> to avoid a wrong connection to NextJS

haha, I haven't thought about that, even the name of NextJS sucks :)